```
$ git commit -m "Testing hooks"
black....................................................................Failed
- hook id: black
- files were modified by this hook

reformatted leaguecog\blitzcrank.py
reformatted leaguecog\leaguecog.py

All done! \u2728 \U0001f370 \u2728
2 files reformatted, 1 file left unchanged.

flake8...................................................................Failed
- hook id: flake8
- exit code: 1

leaguecog/leaguecog.py:173:9: F841 local variable 'ser' is assigned to but never used

don't commit to branch...................................................Passed
print-msg................................................................Passed
- hook id: print-msg
- duration: 0.04s

uses black styling, flake8 linting, and branch name must begin with bugfix/ or feature/
 leaguecog/blitzcrank.py leaguecog/ezreal.py leaguecog/leaguecog.py

```

--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -5,7 +5,7 @@
         rev: 22.6.0
         hooks:
           - id: black
-            language_version: python3.8
+            language_version: python3.9
       - repo: https://gitlab.com/pycqa/flake8
         rev: 3.9.2
         hooks:

--- a/leaguecog/blitzcrank.py
+++ b/leaguecog/blitzcrank.py
@@ -70,11 +70,7 @@ class Blitzcrank(MixinMeta):
             "\n\n"
             "Note: These tokens are sensitive and should only be used in a private channel\n"
             "or in DM with the bot.\n"
-        ).format(
-            command="`{}set api league api_key {}`".format(
-                "!", ("<your_riot_api_key_here>")
-            )
-        )
+        ).format(command="`{}set api league api_key {}`".format("!", ("<your_riot_api_key_here>")))
         if self.task:
             self.task.cancel()
         try:
@@ -94,9 +90,7 @@ class Blitzcrank(MixinMeta):
 
     async def update_version(self):
         """This gets the most recent League API version, then updates our local list of champions"""
-        version = await self.simple_get(
-            "https://ddragon.leagueoflegends.com/api/versions.json"
-        )
+        version = await self.simple_get("https://ddragon.leagueoflegends.com/api/versions.json")
         if not self.champ_api_version:
             self.champ_api_version = version[0]
             self.champlist = await self.simple_get(
@@ -124,9 +118,8 @@ class Blitzcrank(MixinMeta):
             #    and send the author a formatted list of available regions
             currTitle = "Invalid Region"
             currType = "invalidRegion"
-            currMsg = (
-                f"Region {region.upper()} not found. Available regions:\n"
-                + ", ".join([r.upper() for r in self.regions.keys()])
+            currMsg = f"Region {region.upper()} not found. Available regions:\n" + ", ".join(
+                [r.upper() for r in self.regions.keys()]
             )
 
         else:
@@ -171,7 +164,9 @@ class Blitzcrank(MixinMeta):
                         currTitle = "Registration Failure"
                         currType = "apiFail"
                         if req.status == 404:
-                            currMsg = f"Summoner '{name}' does not exist in the region {region.upper()}."
+                            currMsg = (
+                                f"Summoner '{name}' does not exist in the region {region.upper()}."
+                            )
                         elif req.status == 401 or req.status == 403:
                             currTitle = "Invalid Token"
                             currType = "apiFail"
@@ -180,9 +175,7 @@ class Blitzcrank(MixinMeta):
                         else:
                             currTitle = "Unexpected Error"
                             currType = "apiFail"
-                            currMsg = (
-                                f"Riot API request failed with status code {req.status}"
-                            )
+                            currMsg = f"Riot API request failed with status code {req.status}"
             else:
                 currTitle = "Invalid Token"
                 currType = "apiFail"
@@ -221,13 +214,9 @@ class Blitzcrank(MixinMeta):
                             except aiohttp.ContentTypeError:
                                 game_data = {}
                             if req.status == 200:
-                                await self.user_in_game(
-                                    member, user_data, game_data, channel
-                                )
+                                await self.user_in_game(member, user_data, game_data, channel)
                             elif req.status == 404:
-                                await self.user_is_not_in_game(
-                                    member, user_data, channel
-                                )
+                                await self.user_is_not_in_game(member, user_data, channel)
                             elif req.status == 401 or req.status == 403:
                                 await self.token_expired_or_missing()
                             else:
@@ -327,8 +316,6 @@ class Blitzcrank(MixinMeta):
         team100 = await self.config.member(member).active_game.get_raw("team100")
         team200 = await self.config.member(member).active_game.get_raw("team200")
         sent_message = await channel.fetch_message(message_id)
-        embed = await self.build_end_game(
-            user_data["summoner_name"], champ_name, team100, team200
-        )
+        embed = await self.build_end_game(user_data["summoner_name"], champ_name, team100, team200)
         await sent_message.edit(embed=embed)
         await self.config.member(member).active_game.clear_raw()

--- a/leaguecog/leaguecog.py
+++ b/leaguecog/leaguecog.py
@@ -166,13 +166,10 @@ class LeagueCog(
         # set up a ReactionPredicate and interpret the response
         #   then index the server (ex. 'na1') and abbreviation (ex. 'na')
         start_adding_reactions(region_msg, region_emojis)
-        region_pred = ReactionPredicate.with_emojis(
-            region_emojis, region_msg, ctx.author
-        )
+        region_pred = ReactionPredicate.with_emojis(region_emojis, region_msg, ctx.author)
         await ctx.bot.wait_for("reaction_add", check=region_pred)
 
         region_idx = region_pred.result
-        ser = [v["ser"] for v in self.regions.values()][region_idx]
         region = [k for k in self.regions.keys()][region_idx]
 
         # set guild region
@@ -244,9 +241,7 @@ class LeagueCog(
 
             # set up a ReactionPredicate and index text_channel_dict based on response
             start_adding_reactions(channel_msg, number_emojis)
-            channel_pred = ReactionPredicate.with_emojis(
-                number_emojis, channel_msg, ctx.author
-            )
+            channel_pred = ReactionPredicate.with_emojis(number_emojis, channel_msg, ctx.author)
             await ctx.bot.wait_for("reaction_add", check=channel_pred)
 
         else:
@@ -316,9 +311,7 @@ class LeagueCog(
 
     @league.command(name="set-summoner")
     @commands.guild_only()
-    async def set_summoner(
-        self, ctx: commands.Context, name: str = "", region: str = None
-    ):
+    async def set_summoner(self, ctx: commands.Context, name: str = "", region: str = None):
         """
         This sets a summoner name to your Discord account.
         Names with spaces must be enclosed in "quotes". Region is optional.

